src/object.d

```diff
@@ -3184,8 +3184,8 @@ bool _xopCmp(in void*, in void*)
     throw new Error("TypeInfo.compare is not implemented");
 }
 
-void __ctfeWrite(T...)(auto ref T) {}
-void __ctfeWriteln(T...)(auto ref T values) { __ctfeWrite(values, "\n"); }
+void __ctfeWrite(const string s) {}
+void __ctfeWriteln(const string s) {}
```

Comment:
`const(char)[]` (or `in char[]`) would be more flexible for the user. A common pattern with string mixins is to append to a `char[]` in a loop. Requiring immutability is unnecessary.